other/verify_image gcpkms.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: verify-image-gcpkms
+  annotations:
+    policies.kyverno.io/title: Verify Image
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/minversion: 1.4.2
```

Comment:
This sets the minimum compatible version of Kyverno. Use of this new gcpkms key was added later, was it not?

Reply:
As I mentioned below, with this [PR](https://github.com/kyverno/kyverno/pull/2607/), we added that support, but it seems not been released yet.